src/ThemeDispatcher.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Form;
+
+use RuntimeException;
+
+use function array_key_exists;
+
+final class ThemeDispatcher
```

Comment:
It doesn't dispatch anything so... maybe a theme factory?

Reply:
`ThemeContainer` ?

Reply:
Better than dispatcher, yes.

Reply:
done